app/views/cloud_volume/new.html.haml

```diff
@@ -1,16 +1,5 @@
-%form#form_div{:name           => "angularForm",
-               'ng-controller' => "cloudVolumeFormController as vm",
-               "ng-show"       => "vm.afterGet",
-               "ng-cloak"      => "",
-               "miq-form"      => true,
-               "model"         => "vm.cloudVolumeModel",
-               "model-copy"    => "vm.modelCopy"}
-  = render :partial => "layouts/flash_msg"
-  .form-horizontal
-    = render :partial => "common_new_edit"
+= react 'CloudVolumeForm'
```

Comment:
As described [here](https://github.com/ManageIQ/manageiq-ui-classic/pull/7399/files#r571059427) you need to pass the `@storage_manager.id` to the form if it's available:
```suggestion
- if @storage_manager
  = react 'CloudVolumeForm', :storageManagerId => @storage_manager.id.to_s
- else
  = react 'CloudVolumeForm'
```